cli/scripts/e2e/e2e.ts

```diff
@@ -334,7 +335,8 @@ function getCachedDirForHash(repo: Monorepo, hash: string): string {
 
 function getCachedLogFilePathForTask(
   cacheDir: string,
+  pathToPackage: string,
   taskName: string
 ): string {
-  return path.join(cacheDir, ".turbo", `turbo-${taskName}.log`);
+  return path.join(cacheDir, pathToPackage, ".turbo", `turbo-${taskName}.log`);
```

Comment:
@gsoltis is this PR and change going to work us into a weird hole in the future? im thinking that we lose flexibility by  storing cache items with their directory structure, but maybe not

Reply:
I guess moving a package will cause a cache miss. That seems fine.

I think we'll always know the path to the package, there's no notion of running `turbo` in a subtree of the repo without knowing anything about the root of the repo.

We should probably validate that the outputs are inside the monorepo if they are no longer going to be inside the corresponding tasks's package. I don't think we would want a future cache hit to make changes outside of the monorepo.

Reply:
True but do we really care about outside of monorepo use case enough to warn against it?

Reply:
I suppose "just don't do that" is ok.